summarizer_server/tests/test_text_rank.py

```diff
@@ -1,17 +1,100 @@
 import unittest
 import summarizer_server.tests.test_utils as utils
+import math
+import pdb
+
+from nltk import tokenize
 from summarizer_server.text_rank import TextRank
 
 
 class TestTextRank(unittest.TestCase):
     def setUp(self):
         self.text_rank = TextRank()
 
-    def test_article1_returns_15_percent_of_sentences(self):
-        article = utils.get_article1_contents()
-        all_top_sentences = self.text_rank.summarize(article, 100)
-        top_15p_sentences = self.text_rank.summarize(article, 15)
-        self.assertEqual(len(all_top_sentences) * 15 // 100, len(top_15p_sentences))
+    def test_process_html(self):
+        article_html = utils.get_article_contents("article1.html")
+        expected_article_text = utils.get_article_contents("article1.txt")
+        article = self.text_rank.process_html(article_html)
+
+        self.assertEqual(
+            "Poll finds Raptors’ playoff run has attracted new fans across Canada",
+            article.title,
+        )
+        self.assertEqual(expected_article_text, article.text)
+        self.assertEqual("en", article.config.get_language())
+
+    def test_summarize_from_html(self):
+        article_html = utils.get_article_contents("article2.html")
+
+        summary = self.text_rank.summarize_from_html(article_html, 15)
+        self.assertTrue(summary)
```

Comment:
Is there a reason you assert true? Is it testing for list not empty?

Reply:
List is not empty and list is not None. Just testing that there are some elements in the list.